### PR TITLE
add button for EULA

### PR DESCRIPTION
**Pull request by tremblerz**
## Description

Adds a screen which asks the user to provide consent on the EULA (it is a requirement from legal team)

## How to test

Uninstall the previous version, rerun the app and 2nd screen of onboarding will ask "Provide Consent"

NOTE: UI/UX has to be improved and actual license content has to be added. This pr will act as a placeholder.

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "launch_done_subheader": "Remember, you can always update your preferences later.",
     "launch_enable_location": "Enable Location",
     "launch_enable_notif": "Enable Notifications",
+    "launch_sign_eula": "Provide Consent",
```
Comment (tstirrat) [inline app/locales/en.json:44]: Provide consent

Use sentence capitalization generally.

File: app/locales/en.json [hunk 2]
```diff
@@ -58,6 +59,10 @@
     "launch_screen4_header": "You're in complete control of your data. Location data is saved on your phone.",
     "launch_screen4_subheader": "",
     "launch_set_up_phone": "Set up my phone",
+    "launch_eula_title": "COVID Safe Paths\n\nMobile App Privacy Policy",
+    "launch_eula_agree": "I Agree",
```
Comment (tstirrat) [inline app/locales/en.json:63]: sentence case here and throughout.

File: app/views/onboarding/Onboarding4.js [hunk 3]
```diff
@@ -17,13 +19,61 @@ import languages from '../../locales/languages';
...
       />
+      <View style={styles.centeredView}>
+        <Modal
+          animationType='slide'
+          transparent={true} // eslint-disable-line react/jsx-boolean-value
+          visible={modalVisible}
+          onRequestClose={() => {
+            setModalVisible(false);
+          }}>
+          <View
```
Comment (tstirrat) [inline app/views/onboarding/Onboarding4.js:39]: It's ok for the Modal activation state to be in this file. But this is a component in its own right - with its own styles and state. 

Make it a new file.

**Comment by matteocrippa** [comment]
@tstirrat Add a watch on this PR, once accepted also other localizations need to have the new keys in the json files

**Comment by elviraux** [comment]
image
new design

**Comment by tstirrat** [comment]
Closing this for now, until UI rework is done. Feel free to re-open, but I think it's probably a day or more from being ready.

Trying to keep PR list small.

**Comment by StartupSam** [comment]
Have started implementing this with the Figma design (screenshot below). I'm thinking of using a markdown renderer for the EULA itself, so it could be modified easily in the repo - would that work for you @tstirrat? (library is react-native-markdown-display)

Screenshot from 2020-04-22 14-04-14

**Pull request closed without merging**